book/02-git-basics/sections/undoing.asc

```diff
@@ -7,7 +7,8 @@ Be careful, because you can't always undo some of these undos.
 This is one of the few areas in Git where you may lose some work if you do it wrong.
 
 One of the common undos takes place when you commit too early and possibly forget to add some files, or you mess up your commit message.
-If you want to try that commit again, you can run commit with the `--amend` option:
+If you want to redo that commit, make the additional changes you forgot, stage them, and commit
+again using the `--amend` option:
```

Comment:
It may seem weird, but we've tried pretty hard to stick to the one-line-equals-one-sentence rule with the prose in this book. It makes the diffs more readable. Can you remove the line-break here?

Reply:
No, that's fine, I appreciate the value of that, will fix and resubmit.